app/population/controllers/edit.js

```diff
@@ -14,6 +14,10 @@ class DetailsController extends FormWizardController {
   setInitialValues(req, res, next) {
     if (req.form.options.fullPath !== req.journeyModel.get('lastVisited')) {
       const values = omit(req.population, ['moves_from', 'moves_to'])
+
+      values.transfersOut = req.population.moves_from?.length
```

Comment:
Is there any reason to omit these and then re-add them under a different name?

I wonder if you can simplify this method to:

```javascript
req.sessionModel.set(req.population)
```

And then just do this in the view:

```nunjucks
      <div class="govuk-form-group">
        <label class="govuk-label govuk-label--s" for="transfers_in">
          {{ t('population::daily_view.labels.transfers_in') }}
        </label>
        <div id="transfers_in">{{  values.moves_to.length }}</div>
      </div>

      <div class="govuk-form-group">
        <label class="govuk-label govuk-label--s" for="transfers_out">
          {{ t('population::daily_view.labels.transfers_out') }}
        </label>
        <div id="transfers_out">{{  values.moves_from.length }}</div>
      </div>
```

Reply:
Ah, I think I was trying to keep the naming of the api properties separate from the UI rendering templates.

But tbh, that horse has probably bolted, so we might as well have less code :)